### DISCUSSION TITLE
Suggestion: add notifications filters `is:issue` and  `is:pull-request`

### DISCUSSION THREAD

**Discussion by ncw** [community/community#58877] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I love [github notifications](https://github.com/notifications) - they help me keep my project organised.

I moved over from Octocat and a feature I miss from Octocat is being able to filter the list by just issues only or by just PRs.

As far as I can see you can only filter the list by issues OR PRs using `is:issue-or-pull-request`. Ideally we'd have  `is:issue` and  `is:pull-request` filters too.

This would fit much better in my work flow - I set aside specific times for reviewing PRs which take quality concentration, whereas I can go through issues much quicker. I need the filter because I have many of each!

![image](https://github.com/community/community/assets/536803/39b32491-b60f-4787-80c4-789fdded8b74)

I'd be delighted to be proved wrong if I've missed something.

**Comment by pboushy** [comment]
I'm amazed these two were lumped together actually.
I want to make sure I'm reviewing PRs quickly. It's be great to be able to filter this to pull-requests only.
